runtime/Cargo.toml

```diff
@@ -30,9 +30,15 @@ pbkdf2 = "0.12.2"
 hmac = "0.12.1"
 sha2 = "0.10.8"
 rcgen = { version = "0.13.2", features = ["pem"] }
+file-format = "0.26.0"
+calamine = "0.22"
+pdfium-render = "0.8.27"
 
 # Fixes security vulnerability downstream, where the upstream is not fixed yet:
 url = "2.5"
+async-stream = "0.3"
+futures = "0.3"
+tokio-stream = "0.1"
```

Comment:
These three packages probably shouldn't be listed under "Fixes security vulnerability," or should they?